langchain/src/callbacks/base.ts

```diff
@@ -350,27 +350,35 @@ export class CallbackManager extends BaseCallbackManager {
 }
 
 export class ConsoleCallbackHandler extends BaseCallbackHandler {
+  async handleLLMStart(llm: { name: string }, prompts: string[]) {
```

Comment:
not sure if intentional or was just for testing, but flagging that this changed

Reply:
Yea that was on purpose, seemed a bit strange that our console callback handler only implements some methods? But can revert if that's on purpose for some reason

Reply:
I think we'll want to keep as is otherwise there will be too much output during agent runs

Reply:
Alright, undid these changes